features/enlinkd/pom.xml

```diff
@@ -17,5 +17,7 @@
     <module>adapters</module>
     <module>daemon</module>
     <module>tests</module>
+    <module>org.opennms.features.enlinkd.shell</module>
+    <module>org.opennms.features.enlinkd.topogen</module>
```

Comment:
I prefer `shell` and `topogen` here to be consistent with the other module names.
Can we find a better naming as `topogen`? Maybe `generator` or `generation`?

Reply:
fixed to generator